### ISSUE TITLE
Show all owned tokens by default. 

### ISSUE THREAD

**Issue by zreba** [greymass/anchor#1045]
Instead of adding all tokens manually via the 'Add custom token' feature, where you need to know the contract name and token name, show the tokens and balances by default. 

If I go to blocks explorer like bloks.io, I can see all my tokens, I'd expect to see the same in the Wallet.

**Comment by aaroncox** [comment]
The difficulty here comes in how a wallet like Anchor is decentralized, where as a block explorer like bloks.io is running on a centralized server. Bloks can pick and choose to use any API they want and everyone will just use those API servers, where as Anchor has configurable API servers that the user configure. 

Bloks can display all of these tokens because it's using non-native blockchain APIs to retrieve that data. Anchor cannot as easily, because it uses native blockchain APIs to retrieve all of its data.

One of the principals we designed Anchor with is privacy, specifically in that you have complete control over which APIs the wallet will utilize to both read and submit data. While I agree that this feature would be nice to have - it isn't possible unless we sacrifice this aspect of privacy and force Anchor to use one of these non-native blockchain APIs.

I'll keep this issue open since it's something I think we should have, but we need a native solution before we can implement it.

**Comment by maxocoin** [comment]
What about an easy way to open an account in a block explorer? 
A link in the last column to the bloks.io   account page, or any block explorer defined in the setup page
